### PR TITLE
Switch from arangodb to postgres-sql using gorm

### LINKED ISSUE #39: Stop using this shit of arangodb
Switch to the old good postgres SQL with GORM

### PR DESCRIPTION
**Pull request by Brawdunoir**

### CODE REVIEW

**Comment by Brawdunoir** [comment]
Don’t merge it now as we will need to do some update to the GCP docker-compose

File: database/db.go [hunk 1]
```diff
@@ -1,123 +1,37 @@
...
-	}
-
-	// time.Sleep(time.Second * 10)
-	// Wait for the database to be ready
-	_, err = client.Version(context.TODO())
-	for err != nil {
-		fmt.Printf("Waiting for the database to be ready\n")
-		time.Sleep(time.Second * 5)
-		_, err = client.Version(context.TODO())
+		log.Fatal("failed to connect database: ", err)
```
Comment (WhiteDiamondz) [inline database/db.go]:
```suggestion
		log.Fatal("Failed to connect to the database: ", err)
```

File: database/db.go [hunk 2]
```diff
@@ -1,123 +1,37 @@
...
-func GetDatabase(name string) (db driver.Database) {
-	client := GetClient()
-
-	// Check if the database exists and create it if it does not
-	dbExists, err := client.DatabaseExists(context.TODO(), name)
+	// nolint:errcheck
+	db.AutoMigrate(&models.User{}, &models.Room{})
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("failed to migrate database: ", err)
```
Comment (WhiteDiamondz) [inline database/db.go]:
```suggestion
		log.Fatal("Failed to migrate database: ", err)
```

File: tests/user_test.go [hunk 3]
```diff
@@ -64,16 +66,17 @@ func TestUpdateUser(t *testing.T) {
...
-			{Name: "Empty username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":""}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Nil username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":nil}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Integer username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":1}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Object username value", Request: utils.Request{Method: method, URL: userURL, Body: `{"username":{"somekey":"somevalue"}}`}, ResponseCode: http.StatusBadRequest, ResponseBodyRegex: `{"error":".+"}`},
-			{Name: "Not found", Request: utils.Request{Method: method, URL: userURL + "0", Body: `{"username":"test2"}`}, ResponseCode: http.StatusNotFound, ResponseBodyRegex: `{"error":"User not found"}`},
+			{Name: "Success", Request: utils.Request{Method: method, Body: `{"username":"test2"}`}, ResponseCode: http.StatusOK, ResponseBodyRegex: `{"user":{.*"username":"test2"}}`},
```
Comment (WhiteDiamondz) [inline tests/user_test.go]: Why did you add a regex .* before the username in the expected response? Are we currently returning another field ?

**Review by WhiteDiamondz — CHANGES_REQUESTED** [review]
Really small changes required and just want to make sure I understand a few thinks before final aproval

File: tests/user_test.go (hunk 3 above)
Comment (Brawdunoir) [reply tests/user_test.go]: No indeed, thanks to the `json:"-"` statement in models, this is not needed anymore.
